controllers/array_action/settings.py

```diff
@@ -5,6 +5,8 @@
 ISCSI_CONNECTIVITY_TYPE = config.connectivity_type.iscsi
 REPLICATION_COPY_TYPE_SYNC = "sync"
 REPLICATION_COPY_TYPE_ASYNC = "async"
+REPLICATION_TYPE_MIRROR = "mirror"
+REPLICATION_TYPE_EAR = "ear"
```

Comment:
where is the use for this?

Reply:
In validate_addons_request and _get_replication_type

Reply:
I can find REPLICATION_TYPE_EAR anywhere..

Reply:
You are right, I just added it for symmetry. Do you want me to remove it?

Reply:
Sorry, my mistake. I am using it here:
    def _get_replication_type(request):
        if servers_settings.PARAMETERS_REPLICATION_POLICY in request.parameters:
            replication_type = array_settings.REPLICATION_TYPE_EAR
        else:
            replication_type = array_settings.REPLICATION_TYPE_MIRROR

        logger.info("replication type is {}".format(replication_type))
        return replication_type